Carbon.IIS/Functions/Set-CIisAppPoolPeriodicRestart.ps1

```diff
@@ -102,13 +117,14 @@ function Set-CIisAppPoolPeriodicRestart
             if( $PSCmdlet.ShouldProcess($target, $action) )
             {
                 Write-Information "$($prefixMsg)$($icon) $($time)"
+                $scheduleChanged = $true
             }
             if( -not $clearedPrefix )
             {
                 $prefixMsg = ' ' * $prefixMsg.Length
                 $clearedPrefix = $true
             }
-    }
+        }
 
         $currentSchedule.Clear()

```

Comment:
`if( $scheduleChanged )` around all this processing as well, to not even modify-in-memory if not changed?

Reply:
Good catch.